query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedCreateMutationInsideCreateSpec.scala

```diff
@@ -9,42 +9,6 @@ import util._
 class NestedCreateMutationInsideCreateSpec extends WordSpecLike with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities = Set(JoinRelationLinksCapability)
 
-  "a P1! to C1! relation should be possible" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.